### PR TITLE
epson-inkjet-printer-escpr2: update to 1.1.24.

### PR DESCRIPTION
**Pull request by ndowens**
Also remove nocross

### CODE REVIEW

File: srcpkgs/epson-inkjet-printer-escpr2/template [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # Template file for 'epson-inkjet-printer-escpr2'
 pkgname=epson-inkjet-printer-escpr2
-version=1.1.22
+version=1.1.24
 revision=1
 archs="x86_64 i686 aarch64 armv7l"
-create_wrksrc=yes
```
Comment (ericonr) [inline srcpkgs/epson-inkjet-printer-escpr2/template:6]: Why did you remove this?

```
➜ ls masterdir-glibc/builddir/
epson-inkjet-printer-escpr2-1.1.24/                lsb-rpm.spec
epson-inkjet-printer-escpr2-1.1.24-1lsb3.2.tar.gz
```

Comment (ndowens) [reply srcpkgs/epson-inkjet-printer-escpr2/template:6]: It doesnt need to create it, it is already $pkgname-$version, but relooking at it, shouldnt even need to tell what wrksrc it is. Will re-verify, but as it was after update it was dir inside dir

Comment (Chocimier) [reply srcpkgs/epson-inkjet-printer-escpr2/template:6]: Without create_wrksrc it buils, but leaves files in builddir. It is because xbps-src only removes $wrksrc after build.

Comment (ndowens) [reply srcpkgs/epson-inkjet-printer-escpr2/template:6]: odd that it needs it anyway but ok

Comment (ndowens) [reply srcpkgs/epson-inkjet-printer-escpr2/template:6]: Fixed, had to add build_wrksrc; if I used wrksrc=.. it kept adding dirs
